automation/src/main/java/org/wso2/testgrid/automation/executor/JMeterResultCollector.java

```diff
@@ -54,6 +59,11 @@ public void sampleOccurred(SampleEvent sampleEvent) {
             TestCaseUOW testCaseUOW = new TestCaseUOW();
             super.sampleOccurred(sampleEvent);
             SampleResult result = sampleEvent.getResult();
+            String samplerData = result.getSamplerData();
+
+            if (samplerData.length() > 5000) {
```

Comment:
Value is hardcoded here, should use the variable `FAILURE_MESSAGE_LIMIT` defined above

Reply:
Fixed in 6d26744